sdk/tests/deqp/functional/gles3/es3fShaderCommonFunctionTests.js

```diff
@@ -843,7 +843,11 @@ goog.scope(function() {
             /** @type {number} */ var sign = Math.abs(rnd.getInt()) & 0x1;
             /** @type {number} */ var value = (sign << 31) | (exp << 23) | mantissa;
 
-            // TODO: check the following assert
+            // Convert int to float.
+            var view = new DataView(new ArrayBuffer(4));
+            view.setUint32(0, value, true);
```

Comment:
Same here as I commented in the other patch.  If the original data is int, let's just use setInt32 instead of setUint32, just to be consistent and clear.